sktime/forecasting/ensemble_algorithms.py

```diff
@@ -0,0 +1,234 @@
+from sklearn.base import BaseEstimator
+from scipy.optimize import nnls
+import numpy as np
+
+
+class EnsembleAlgorithm(BaseEstimator):
+    """Wrapper class to take ensemble algorithms and allow them to be easily set
+
+    Parameters
+    ----------
+    n : float
+    loss_func : function
+    """
+    def __init__(self, n=10, loss_func=None, **kwargs):
+        self.n = n
+        self.weights = np.ones(n)/n
+        self.loss_func = loss_func
+
+    def _predict(self, expert_predictions):
```

Comment:
minor comment - it is convention to precede methods by an underscore if they are private. Most of these though look like public, they are in fact called from outside in the `EnsembleForcaster` templated methods. So they should be without a preceding underscore, no?

Reply:
ahaha, I didn't understand the convention too well, let me fix that.

Reply:
I also kinda just used the format I found in `compose.EnsembleForecaster`. Should I make the predict function then read `predict` or `_predict`?